app/controllers/admin/words_controller.rb

```diff
@@ -0,0 +1,90 @@
+module Admin
+  class WordsController < ApplicationController
```

Comment:
```suggestion
class Admin::WordsController < ApplicationController
```
Viết gọn lại cũng được e nha, xong lùi lại 1 tab với các code bên trong

Reply:
done